zerver/webhooks/appfollow/view.py

```diff
@@ -16,7 +16,11 @@
 def api_appfollow_webhook(request: HttpRequest, user_profile: UserProfile,
                           payload: Dict[str, Any]=REQ(argument_type="body")) -> HttpResponse:
     message = payload["text"]
-    app_name = re.search('\A(.+)', message).group(0)
+    app_name_search= re.search('\A(.+)', message)
+    if app_name_search is None:
+        return json_error(_('Error: Could not find App name'))
```

Comment:
I think we actually want this to raise an exception rather than return a `json_error`; that will end up notifying the right person.  So I think maybe just doing an `assert` here would be better.

Reply:
Amended to simple assert.